src/_base/docker/image/console/root/lib/task/assets/apply.sh.twig

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+function task_assets_apply()
+{
+    local ASSETS_DIR="${ASSETS_DIR:-tools/assets/development}"
+    local IMPORT_COMMAND=""
+
+    if [ "{{ @('database.type')|raw }}" == "mysql" ]; then
```

Comment:
Just a thought here, rather than rendering dead code I think this would be better as either:
a) an environment variable - symfony even requires the variable for it's DBAL platform, so would kill two birds with one stone
b) use twig conditions to render only the relevant task

I prefer (a) really

Reply:
True, could you please open an issue for it?

Reply:
ah, you already opened PR, even better 👍